tests/Nexucis/Tests/Elasticsearch/Helper/Nodowntime/AbstractIndexHelperTest.php

```diff
@@ -24,17 +24,19 @@ abstract class AbstractIndexHelperTest extends TestCase
     /**
      * initialize static data
      */
-    public static function setUpBeforeClass()
+    public static function setUpBeforeClass(): void
     {
-        // load static data
-        self::$documents = json_decode(file_get_contents('http://data.consumerfinance.gov/api/views.json'));
+        self::$documents = self::$documents === null
```

Comment:
definitely better like that indeed. Thanks for changed that !

Reply:
Faster and stable! :-)